src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -9089,53 +9089,48 @@ void VRFTerminalUnitEquipment::ControlVRFToLoad(EnergyPlusData &state,
 
     // set supplemental heating coil calculation if the condition requires
```

Comment:
This next section was refactored to make it easier to interpret what was happening and why. This allowed a simple change to handle when the supp heating should activate and when it should not. This simple change will be highlighted next.